internal/snmp/translate_test.go

```diff
@@ -87,3 +89,54 @@ func TestTrapLookupFail(t *testing.T) {
 		})
 	}
 }
+
+type TestingMibLoader struct {
+	folders []string
+	files   []string
+}
+
+func (t *TestingMibLoader) appendPath(path string) {
+	t.folders = append(t.folders, path)
+}
+
+func (t *TestingMibLoader) loadModule(path string) error {
+	t.files = append(t.files, path)
+	return nil
+}
+func TestFolderLookup(t *testing.T) {
+	paths := [][]string{
+		{"testdata"},
+		{"testdata", "loadMibsFromPath"},
+		{"testdata", "loadMibsFromPath", "linkTarget"},
+		{"testdata", "loadMibsFromPath", "root"},
+		{"testdata", "mibs"},
+	}
+	tests := []struct {
+		name    string
+		folders []string
+		files   []string
+	}{
+		{
+			name:  "loading folders",
+			files: []string{"emptyFile", "testmib"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			loader := TestingMibLoader{}
+
+			err := LoadMibsFromPath([]string{"testdata"}, testutil.Logger{}, &loader)
```

Comment:
I thought we were going to have the test load testdata/loadMibsFromPath/root, not testdata. If we load testdata here, when we modify other tests that use files in testdata, like testdata/mibs, it will affect the paths in this test.

If we load testdata/loadMibsFromPath/root, this test won't be affected when other tests change the contents of testdata outside of testdata/loadMibsFromPath.

Also, this path should be in the table along with files and paths